DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Imports:
     emo,
     nycflights13,
     tufte,
-    tidyverse
+    tidyverse,
+    nycflights13
```

Comment:
This is already included a few lines above. Please remove the duplicate. I don't think it will actually break anything, but it's unnecessary.

Reply:
I just fixed this manually 😁